release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2127,6 +2127,28 @@ This behavior has been fixed, and the references are now accepted and correctly
 [id="ocp-4-14-olm-bug-fixes"]
 ==== Operator Lifecycle Manager (OLM)
 
+* Before {product-title} 4.14.2, Operator Lifecycle Manager (OLM) could overwrite privileged cluster roles used by {product-title} and Kubernetes resources if you created an Operator group with the same name. In {product-title} 4.14.2 and later, if you create an Operator group that conflicts with a privileged cluster role, OLM renames the cluster role and adds a hash value to avoid conflicts. OLM renames only Operator groups that conflict with a privileged cluster role. You must ensure that the name of your Operator group does not conflict with other cluster roles that exist on the cluster.
```

Comment:
>In {product-title} 4.14.2 and later, if you create an Operator group that conflicts with a privileged cluster role, OLM renames the cluster role and adds a hash value to avoid conflicts.

It's a little bit confusing. OLM doesn't **rename** any `cluster role` anymore. It just generates a ClusterRole name in the form of: `olm.og.<operator_group_name>.<admin_edit_or_view>-<hash_value>` to avoid conflict.

Reply:
Thank you for explaining @jianzhangbjz! Addressed in https://github.com/openshift/openshift-docs/pull/67891/commits/c467cee541c15e29cc44569b81c98c8b6b4da8a1

Note, I moved the release note to the OCP 4.14.2 bug fixes.